include/deal.II/base/tensor.h

```diff
@@ -893,14 +893,69 @@ operator Tensor<1,dim,Tensor<rank_-1,dim,OtherNumber> > () const
 }
 
 
+
+namespace internal
+{
+  namespace TensorSubscriptor
+  {
+    namespace
+    {
+      template <typename ArrayElementType, int dim>
+      ArrayElementType &
+      subscript (ArrayElementType *values,
+                 const unsigned int i,
+                 dealii::internal::int2type<dim>)
+      {
+        Assert (i<dim, ExcIndexRange(i, 0, dim));
+        return values[i];
+      }
+
+
+      template <typename ArrayElementType, int dim>
+      const ArrayElementType &
+      subscript (const ArrayElementType *values,
+                 const unsigned int i,
+                 dealii::internal::int2type<dim>)
+      {
+        Assert (i<dim, ExcIndexRange(i, 0, dim));
+        return values[i];
+      }
```

Comment:
Yes -- leftover from rounds of otherwise unsuccessful attempts. Will take care of it.